_posts/en/newsletters/2024-12-13-newsletter.md

```diff
@@ -0,0 +1,308 @@
+  lower capacity channels would exhaust a party's balance.  For two
+  participants, as in current-generation LN channels, each additional
+  satoshi given to capacity will increase the range of the wealth
+  distribution by one.  However, in [channel factories][topic channel
+  factories] and other multiparty constructions that allow funds to move
+  offchain between _k_ parties, each additional sat given to capacity
+  increases the range of wealth distributions by one _for each of the
+  k parties_---greatly increasing the number of feasible payments and
+  reducing the risk of depletion.
+
+  For example, consider an example of the current LN where Alice has
+  channels with Bob and Carol, who also each have a channel with each
+  other: {AB, AC, BC}.  Each channel has a capacity of 1 BTC.  In this
+  configuration, the maximum value Alice can control (and thus send or
+  receive) is 2 BTC.  The same limit applies to Bob and Carol.  The same
+  limits would also apply if the total 3 BTC was used to recreate all
+  three channels in a channel factory; however, if the factory
+  remained operational, an offchain state update between all three
+  parties could allow Alice to control up to the full 3 BTC, and thus
+  she could send or receive up to 3 BTC (the same for Bob and Carol).
+  This use of multiparty offchain protocols allows the same amount of
+  capital to give each participant access to higher-capacity channels
+  that are less likely to become depleted.  Fewer depletions and a
+  greater range of feasible payments correspond to a greater maximum
+  throughput of LN, as Pickhardt has written about before (see
+  Newsletters [#309][news309 feasible] and [#325][news325 feasible]).
+
+  Both in his post and in the Optech Deep Dive discussion, Pickhardt
+  sought data (e.g., from large LSPs) that could be used to help
+  validate the simulated results.
+
+- **Poll of opinions about covenant proposals:** /dev/fd0 [posted][fd0
+  poll] to the Bitcoin-Dev mailing list a link to a [public poll][wiki
+  poll] of developer opinions about selected covenant proposals.  Yuval
+  Kogman [noted][kogman poll] that developers were asked to evaluate
+  both the "technical merit of [each] proposal" and their "vibes-based
+  opinion about [its] community support", but without developers being
+  able to express all possible combinations of the two through the
+  poll's limited options.  Jonas Nick [requested][nick poll] "separating
+  technical evaluations from community support" and Anthony Towns
+  [suggested][towns poll] simply asking developers whether they had any
+  outstanding concerns about each proposal.  Both Nick and Towns
+  recommended developers link to evidence and arguments supporting their
+  opinions.
+
+  Although faults in the poll were highlighted, a showing of greater
+  support for some proposals over other proposals may help covenant
+  researchers converge on a short list of ideas for the broader
+  community to review.
```

Comment:
> I think the table already has enough options and I don't want to confuse everyone with all possible combinations

Anyone who can't handle a poll with nine options is not qualified to comment on technical merit of consensus changes.

>  This has been answered on [mailing list](https://groups.google.com/g/bitcoindev/c/fdxkE1Al4TI/m/O4NNSbRXAgAJ) and there are enough options that a developer could evaluate a propose technically.

Being able to evaluate a proposal technically is not the same as separating technical concerns from community support concerns, especially given Kogman's point about not all states being reachable.

> This is possible with the current table

Yes, but (as with the point above) people who choose to do this may be at a perceived disadvantage to people who select from all available choices.

> These aren't really faults but suggestions and have been answered.

I believe all of these criticisms were meant as identifications of faults, and none of your answers completely satisfied the original criticism.

> did not mention the changes that were made based on feedback

These are all definitely improvements, thank you for making them.  I wasn't aware of some of those changes (they happened before I first looked at the table).  However, I don't think readers of the newsletter would benefit from learning about any of those past changes, so I don't see a need to mention them.

> the primary goal of getting a rough idea about opinion of different developers is achievable with the table.

That's what we say in the conclusion of this news item, although I encourage this table to mainly be used by covenant devs to focus their work.  I don't think the table is useful for non-cov devs or the general public due to the table's many problems and the lack of response from many of the experts we would expect to be involved in an actual soft fork.